build/teamcity-check.sh

```diff
@@ -35,22 +37,23 @@ fi
 tc_start_block "Ensure generated code is up-to-date"
 # Buffer noisy output and only print it on failure.
 run build/builder.sh make generate &> artifacts/generate.log || (cat artifacts/generate.log && false)
-run build/builder.sh make buildshort &> artifacts/buildshort.log || (cat artifacts/buildshort.log && false)
 rm artifacts/generate.log
+check_clean "Run \`make generate\` to automatically fix these. For only BUILD.bazel issues, run \`bazel run //:gazelle\`."
```

Comment:
I'd also be fine with suggesting `make bazel-generate` here.

Reply:
have to teach people to stop using make somehow :P